cmd/esc/cli/cli_test.go

```diff
@@ -237,7 +237,8 @@ type testEnvironments struct {
 }
 
 func (e *testEnvironments) LoadEnvironment(ctx context.Context, envName string) ([]byte, eval.Decrypter, error) {
```

Comment:
@pgavlin do you know why this method only receives environment name? The corresponding method in pulumi-service receives a ref string which includes `project/env` and evaluates correctly, but not sure what this test client is doing differently. maybe I'm misunderstanding something?

Reply:
`envName` should include the entire ref string here. what behavior are you observing? have you attached a debugger and stepped through the failing tests?